packages/graphql/tests/integration/aggregations/top_level/bigint.test.ts

```diff
@@ -0,0 +1,288 @@
+import { generate } from "randomstring";
+import neo4j from "../../neo4j";
+import { Neo4jGraphQL } from "../../../../src/classes";
+
+describe("aggregations-top_level-bigint", () => {
+    let driver: Driver;
+
+    beforeAll(async () => {
+        driver = await neo4j();
+    });
+
+    afterAll(async () => {
+        await driver.close();
+    });
+
+    test("should return the min of node properties", async () => {
+        const session = driver.session();
+
+        const typeDefs = `
+            type Movie {
+                id: ID
+                imdbRatingBigInt: BigInt
+            }
+        `;
+
+        const id = generate({
+            charset: "alphabetic",
+            readable: true,
+        });
+
+        const neoSchema = new Neo4jGraphQL({ typeDefs });
+
+        try {
+            await session.run(
+                `
+                    CREATE (:Movie {id: $id, imdbRatingBigInt: 1.1})
+                    CREATE (:Movie {id: $id, imdbRatingBigInt: 2.1})
+                    CREATE (:Movie {id: $id, imdbRatingBigInt: 3.1})
+                    CREATE (:Movie {id: $id, imdbRatingBigInt: 4.1})
+                `,
+                {
+                    id,
+                }
+            );
+
+            const query = `
+                {
+                    moviesAggregate(where: {id: "${id}"}) {
+                        imdbRatingBigInt {
+                            min
+                        }
+                    }
+                }
+            `;
+
+            const gqlResult = await graphql({
+                schema: neoSchema.schema,
+                source: query,
+                contextValue: { driver, driverConfig: { bookmarks: [session.lastBookmark()] } },
+            });
+
+            if (gqlResult.errors) {
+                console.log(JSON.stringify(gqlResult.errors, null, 2));
+            }
+
+            expect(gqlResult.errors).toBeUndefined();
+
+            expect((gqlResult.data as any)[`moviesAggregate`]).toEqual({
+                imdbRatingBigInt: {
+                    min: "1.1",
+                },
+            });
+        } finally {
+            await session.close();
+        }
+    });
+
+    test("should return the max of node properties", async () => {
+        const session = driver.session();
+
+        const typeDefs = `
+            type Movie {
+                id: ID
+                imdbRatingBigInt: BigInt
+            }
+        `;
+
+        const id = generate({
+            charset: "alphabetic",
+            readable: true,
+        });
+
+        const neoSchema = new Neo4jGraphQL({ typeDefs });
+
+        try {
+            await session.run(
+                `
+                    CREATE (:Movie {id: $id, imdbRatingBigInt: 1.1})
+                    CREATE (:Movie {id: $id, imdbRatingBigInt: 2.1})
+                    CREATE (:Movie {id: $id, imdbRatingBigInt: 3.1})
+                    CREATE (:Movie {id: $id, imdbRatingBigInt: 4.1})
```

Comment:
Just a little confusing / bad practice using a shared `ID` field here.
I understand the intent, but how about using a String field or something instead?

Reply:
Good catch I've made this change.